### PR TITLE
Vent board fixes

### PR DESCRIPTION
**Pull request by AlexTheDragon8**
- Switched U3 and U4's sources and drains
- Connected R13 and R18 to +5VBAT (instead of +BATT)

### CODE REVIEW

File: vent/vent.kicad_pcb [hunk 1]
```diff
@@ -194,180 +187,49 @@
...
-    (add_net "Net-(U3-Pad5)")
-    (add_net "Net-(U3-Pad6)")
-    (add_net "Net-(U3-Pad7)")
     (add_net "Net-(U4-Pad1)")
     (add_net "Net-(U6-Pad1)")
-    (add_net "Net-(U9-Pad4)")
-    (add_net "Net-(U9-Pad5)")
   )
 
-  (module Team_Logo:Logo_SilkScreen_10mm (layer F.Cu) (tedit 0) (tstamp 5BEFD6E1)
```
Comment (akmorrison) [inline vent/vent.kicad_pcb]: Team Logo was deleted

File: vent/vent.kicad_pcb [hunk 2]
```diff
@@ -436,100 +298,6 @@
       (net 35 /Microcontroller/~MCLR))
   )
 
-  (module Housings_SOIC:SOIC-28W_7.5x17.9mm_Pitch1.27mm (layer F.Cu) (tedit 58CC8F64) (tstamp 5C1100CA)
```
Comment (akmorrison) [inline vent/vent.kicad_pcb]: microcontroller is gone

File: vent/vent.kicad_pcb [hunk 3]
```diff
@@ -931,114 +699,6 @@
     )
   )
 
-  (module Housings_SOIC:SO-8_5.3x6.2mm_Pitch1.27mm (layer F.Cu) (tedit 59920130) (tstamp 5C25EEAE)
```
Comment (akmorrison) [inline vent/vent.kicad_pcb]: MCP2562 is deleted

**Comment by AlexTheDragon8** [comment]
I redid those changes based on the `master` branch. The symbol libraries I have were downloaded from the `kicad-symbols` Github repository on Oct. 12. I have no idea why KiCAD is changing the library files. For now, should I just not add those changes to the commit?

**Comment by amihaila** [comment]
Closed, work is in `vent_brd_fixes_2`

**Pull request closed without merging**